Assets/MixedRealityToolkit.SDK/Profiles/DefaultMixedRealityGlobalInputEventSystemProfile.asset

```diff
@@ -0,0 +1,15 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!114 &11400000
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 0}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: 0b7fd900dbf72e340b2bc70ee7cec819, type: 3}
+  m_Name: MixedRealityGlobalInputEventSystemProfile
+  m_EditorClassIdentifier: 
+  isCustomProfile: 1
```

Comment:
Why have a profile if no configurable data is stored in it?

Reply:
As I explained above, the profile here is simply a template. 

This profile is meant to be extensible, and as such, a child implementation can expose configurable data to their app.